scripts/util/metadata/field.rb

```diff
@@ -101,7 +101,7 @@ def initialize(hash)
     end
 
     if wildcard? && !object?
-      if !@examples.any? { |example| example.is_a?(Hash) }
+      if @examples.any? { |example| !example.is_a?(Hash) }
```

Comment:
I'm pretty sure this isn't wanted

Reply:
Yeah, I was really not sure what the deal is here. I hit it when `@examples` is an empty array so `![].any? { ... } == true`. The error seemed to be saying these need to be hashes, which didn't seem like it should fire on an empty array. But I could definitely be wrong.

Reply:
@binarylogic could you weigh in here? Happy to undo this if there's a better way to avoid the error.